Assets/Scripts/StreetBuilder.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public sealed class StreetBuilder : MonoBehaviour {
+
+	//VARIAÇÕES
+	public GameObject cityStreet;
+	public GameObject countryStreet;
+	public GameObject highwayStreet;
+	private GameObject mainStreet;
+
+	//CONTROLADOR DE REPETIÇÕES DE PISTA
+	[SerializeField]
+	private int timeChange = 3; 
+	private int timeChangeCounter = 0;
+
+	public static bool isDestroyed = false;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://www.codacy.com/app/mendes-barreto/BSToysGame/pullRequest?prid=684733)